packages/buefy-next/src/components/carousel/Carousel.vue

```diff
@@ -228,6 +228,12 @@ export default {
         },
         hasNext() {
             return this.repeat || this.activeChild < this.childItems.length - 1
+        },
+
+        // internal index of the active child
```

Comment:
Could this comment be removed? The comment seems to repeat the method name and should be removed. See: https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/

Reply:
I will remove it!